src/wp-includes/cache-compat.php

```diff
@@ -199,3 +199,116 @@ function wp_cache_supports( $feature ) {
 		return false;
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_get_salted' ) ) :
+	/**
+	 * Retrieves cached query data if valid and unchanged.
```

Comment:
Some obsolete mention of query specific data in here.
```suggestion
	 * Retrieves cached data if valid and unchanged.
```